app/Http/Controllers/Assisted/AssistedFamilyIncomeUpdate.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers\Assisted;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\FamilyIncome\UpdateRequest;
+use App\Models\Assisted;
+
+class AssistedFamilyIncomeUpdate extends Controller
+{
+    /**
+     * @param UpdateRequest $request
+     * @param Assisted $assisted
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function __invoke(UpdateRequest $request, Assisted $assisted)
+    {
+        $assisted->social_programs = $request->social_programs == "" ? 0.00 : $request->social_programs;
+        $assisted->social_security_contribution = $request->social_security_contribution == "" ? 0.00 : $request->social_security_contribution;
+        $assisted->income_tax = $request->income_tax == "" ? 0.00 : $request->income_tax;
+        $assisted->alimony = $request->alimony == "" ? 0.00 : $request->alimony;
+        $assisted->extraordinary_expenses = $request->extraordinary_expenses == "" ? 0.00 : $request->extraordinary_expenses;
```

Comment:
Você pode fazer uma função para retornar o valor zerado caso for uma string vazia ou usar Laravel Mutators no model do assistido, nesses campos.